src/status_im/contexts/profile/push_notifications/events.cljs

```diff
@@ -193,3 +192,40 @@
             (save-profile-setting-fx setting))]}))
 
 (rf/reg-event-fx :notifications/news-notifications-switch news-notifications-switch)
+
+(defn check-notifications-blocked
+  [_]
+  {:fx [[:effects/check-notifications-permissions
+         {:on-success [:notifications/determine-notifications-blocked]
+          :on-error   [:log/debug "failed to check notification permissions"]}]]})
+
+(rf/reg-event-fx :notifications/check-notifications-blocked check-notifications-blocked)
+
+(defn determine-notifications-blocked
```

Comment:
This event very likely won't be properly tested by a blackbox approach from QA, if that's even practical. A unit test for this event can cover this blind spot.